index.bs

```diff
@@ -2975,44 +5227,121 @@ partial interface MLGraphBuilder {
   MLOperand squeeze(MLOperand input, optional MLSqueezeOptions options = {});
 };
 </script>
-<div algorithm=squeeze>
+
+<div>
     **Arguments:**
         - *input*: an {{MLOperand}}. The input tensor.
         - *options*: an optional {{MLSqueezeOptions}}. The optional parameters of the operation.
-            - *axes*: a sequence of {{unsigned long}}. Indices to the shape dimensions of size 1 to eliminate. The values in the sequence must be in the range [0, N-1] where N is the rank of input tensor. When not specified, every shape dimensions of size 1 in the tensor are eliminated.
 
     **Returns:** an {{MLOperand}}. The output tensor of the same or reduced rank with the shape dimensions of size 1 eliminated.
 </div>
 
-### The tanh() method ### {#api-mlgraphbuilder-tanh}
+{{MLSqueezeOptions}} has the following members:
+<dl dfn-type=dict-member dfn-for=MLSqueezeOptions>
+    : <dfn>axes</dfn>
+    ::
+        A sequence of {{unsigned long}}.
+        Specifies the indices to the shape dimensions of size 1 to eliminate. The values in the sequence must be in the range [0, N-1] where N is the [=rank=] of the input tensor.
+        When not specified, every shape dimensions of size 1 in the tensor are eliminated.
+</dl>
+
+<details open>
+  <summary>
+    The {{MLGraphBuilder/squeeze(input, options)}} steps are:
+  </summary>
+  <div algorithm=squeeze class=algorithm-steps>
+    1. [=Assert=]: the type of |input| is {{MLOperand}}.
+    1. If |options|.{{MLSqueezeOptions/axes}} [=map/exists=], then:
```

Comment:
You may also want to check the size of axes, it should be smaller than the rank of input tensor.